source/fundamentals/builders/sort.txt

```diff
@@ -60,8 +60,9 @@ The ``Sorts`` class is a builder that provides static factory methods for all so
 criteria operators supported by MongoDB. These methods return a ``Bson`` object
 that you can pass to the  ``sort()`` method of a ``FindFlow`` instance or to
 ``Aggregates.sort()``.
-If you want to learn more about the ``Aggregates``
-class, see our :doc:`guide on the Aggregates builder </fundamentals/builders/aggregates>`.
+
+To learn more about the ``Aggregates``
+class, see the :ref:`guide on the Aggregates builder <aggregates-builders>`.
```

Comment:
Non-blocking suggestion - I think this is more in line with the majority of the other pages where we have links like this:

```suggestion
class, see the :ref:`Aggregates builder <aggregates-builders>` guide.
```

Reply:
Will update throughout on both pages.